Source/JavaScriptCore/Scripts/generate-combined-inspector-json.py

```diff
@@ -29,12 +29,12 @@
 import re
 import sys
 
-if len(sys.argv) < 2:
-    print("usage: %s [json files or directory of json files ...]" % os.path.basename(sys.argv[0]))
+if len(sys.argv) < 3:
+    print("usage: %s [json files or directory of json files ...] 'flags'" % os.path.basename(sys.argv[0]))
```

Comment:
Might be better to name this `conditional-flags` instead of just `flags` which sound like switches/options.